source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -25,21 +25,43 @@ export function appendChild(node: Node, child: Node) {
 }
 
 export function setProperty(target: any, propertyName: string, propertyValue: any, defaultValue: any = undefined) {
-	if (propertyValue && (!defaultValue || defaultValue !== propertyValue)) {
+    let targetIsAlreadySet = target[propertyName] !== undefined;
+
+    if (propertyValue === undefined || propertyValue === null) {
+        if (!targetIsAlreadySet) {
+            delete target[propertyName];
+        }
+    }
+    else if (propertyValue !== defaultValue) {
 		target[propertyName] = propertyValue;
```

Comment:
>	 [](start = 0, length = 1)

Nit: Pls replace tab with spaces (one-time config in your source editor settings);

Reply:
My environment is already setup the right way, yet this keeps happening. Honestly I don't think it's a big deal. I fixed it this time.